andela/selene-ah-frontend
# #163218473 Implement the 404 page

#### What does this PR do?
Adds a 404 page to the application
  #### Description of Task to be completed?
When a user navigates to an undefined route, he should be redirected to the 404 page
  #### How should this be manually tested?
 Pull this branch
Navigate to a non-registered route like `no-match`
The 404 page should show
  #### Any background context you want to provide?
Nil
  #### What are the relevant pivotal tracker stories?
[#163218473](https://www.pivotaltracker.com/story/show/163218473)
  #### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2019-01-15 at 10 34 11 am" src="https://user-images.githubusercontent.com/28215750/51171410-21fcbb80-18b1-11e9-9822-1ee910b88518.png">

  #### Questions:
